source/guides/1.1/getting-started.md

```diff
@@ -768,7 +765,7 @@ describe Web::Controllers::Books::Create do
 end
 ```
 
-Making these tests pass is easy enough.
+Let's make these tests pass!
```

Comment:
In my experience it is not a good idea to tell someone that something is "easy enough". What if the reader did not understand it? They might feel pretty stupid when they are told that this is supposed to be easy.